app/views/users/noticed_notifications/index.html.erb

```diff
@@ -11,12 +11,21 @@
       <span id="all-notifications" class="active notifications-tab" data-action="click->notifications#activeAllNotifications"><%= t("users.notifications.all") %></span>
       <span id="unread-notifications" class="notifications-tab" data-action="click->notifications#activeUnreadNotifications"><%= t("users.notifications.unread") %></span>
     </div>
+    <div>
     <% if @unread.count.positive? %>
       <div>
         <%= link_to t("users.notifications.mark_as_read_button"), mark_all_as_read_url(current_user), :method=> :patch, class: "anchor-text" %>
       </div>
     <% end %>
   </div>
+    <div>
+    <% if @notification.count.positive? %>
+      <div>
+        <%= link_to t("users.notifications.clear_all"), clear_all_url(current_user), class: "primary-delete-button", data: {toggle: "modal", target: "#clear_all"} %>
+      </div>
+  <% end %>
+    </div>
```

Comment:
```suggestion
    <div>
      <% if @unread.count.positive? %>
        <%= link_to t("users.notifications.mark_as_read_button"), mark_all_as_read_url(current_user), :method=> :patch, class: "anchor-text" %>
      <% end %>
      <% if @notification.count.positive? %>
        <%= link_to '#', data: {toggle: "modal", target: "#clear_all_modal_confirmation" }, class: "anchor-text-danger" do %>
          <span><%= t("users.notifications.clear_all") %></span>
        <% end %>
      <% end %>
    </div>
```

Reply:
Always try to make formattable code, that increases code readability!

I have added a new class `anchor-text-danger`.
Use this style for that:

```
.anchor-text-danger {
  color: #ff5959;
  text-decoration: underline;
  margin-left: 5px;
}
```

Reply:
Thanks, will keep that in mind.